pkgs/data/fonts/dejavu-fonts/default.nix

```diff
@@ -1,36 +1,23 @@
-{fetchurl, stdenv, fontforge, perl, fontconfig, FontTTF}:
+{fetchFromGitHub, stdenv, fontforge, perl, FontTTF}:
 
-let version = "2.35" ; in
+let version = "2.37" ; in
 
 stdenv.mkDerivation rec {
   name = "dejavu-fonts-${version}";
-  #fontconfig is needed only for fc-lang (?)
   buildInputs = [fontforge perl FontTTF];
 
-  unicodeData = fetchurl {
-    url = http://www.unicode.org/Public/6.1.0/ucd/UnicodeData.txt ;
-    sha256 = "1bd6zkzvxfnifrn5nh171ywk7q56sgk8gdvdn43z9i53hljjcrih";
-  };
-  blocks = fetchurl {
-    url = http://www.unicode.org/Public/6.1.0/ucd/Blocks.txt;
-    sha256 = "0w0vkb09nrlc6mrhqyl9npszdi828afgvhvlb1vs5smjv3h8y3dz";
-  };
-
-  src = fetchurl {
-    url = "mirror://sourceforge/dejavu/dejavu-fonts-${version}.tar.bz2";
-    sha256 = "1xdbi4llrq1qbkd73352ibrfqcbz93dww8hab216qz5szd95yvv4";
+  src = fetchFromGitHub {
+    owner = "dejavu-fonts";
+    repo = "dejavu-fonts";
+    rev = "0eda8a319c08835009849583cd090bb5b141ce25";
```

Comment:
I think it would be more intuitive to have
```
    rev = "version_2_37";
```
here. Or perhaps even
```
    rev = "version_${stdenv.lib.replaceStrings ["."] ["_"] version}";
```